### PR TITLE
refactor: use cookies

### PR DESCRIPTION
**Pull request by llorllale**
Signed-off-by: George Aristy <george.aristy@securekey.com>

### CODE REVIEW

File: pkg/restapi/operation/operations.go [hunk 1]
```diff
@@ -17,6 +17,8 @@ import (
 	"net/http"
 	"net/url"
 
+	"github.com/trustbloc/hub-auth/pkg/restapi/common/store/cookie"
+
```
Comment (Baha-sk) [inline pkg/restapi/operation/operations.go:21]: can this be grouped with `hub-auth` imports below?

File: pkg/restapi/operation/operations_test.go [hunk 2]
```diff
@@ -26,6 +26,8 @@ import (
 	"testing"
 	"time"
 
+	"github.com/trustbloc/hub-auth/pkg/restapi/common/store/cookie"
```
Comment (Baha-sk) [inline pkg/restapi/operation/operations_test.go:29]: same here

**Review by fqutishat — APPROVED** [review]

File: pkg/restapi/operation/operations.go (hunk 1 above)
Comment (llorllale) [reply pkg/restapi/operation/operations.go:21]: will be done in #51

File: pkg/restapi/operation/operations_test.go (hunk 2 above)
Comment (llorllale) [reply pkg/restapi/operation/operations_test.go:29]: will be done in #51

**Review by Baha-sk — APPROVED** [review]
lgtm.. only imports need to be properly grouped

**Pull request merged**